hospitals/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.0.6 on 2020-05-28 02:39
+# Generated by Django 3.0.7 on 2020-07-26 03:24
```

Comment:
How will our existing databases react to changing an initial migration?

Reply:
I think it should be fine --- unless, is there any data currently in our database that we want to keep?

Reply:
@kaashmonee I guess I've just fallen into this trap. I had to drop my database and recreate it from scratch, and given the fact that we don't have any script to create a hospital, and the procedure is not documented, it was a rough ride. We should pay attention not to break our own local development environments in the future.